### PR TITLE
Make Opening hours more consistent

### LINKED ISSUE #469: A missing unified view for the Öffnungszeiten
The view of the Öffnungszeiten varies a lot between several POIs.
The view should be unified.

ex. POI: Parking place
image

ex. POI: Rund um's Fahrrad
image

ex. POI: Nette Toilette
image

QA system version:
 image: mfdz/opentripplanner:otp2-15fd09f6e08683d05cfe96e1400a33356d17afa9
 image: stadtnavi/digitransit-ui:b10a74a97df348e1962ffc454571053966332520
 image: stadtnavi/hsl-map-server:f386dcbf8ea8d9f99a9e7772e0324bed2e20fb2a
 image: stadtnavi/fares-service:b80f1ce50fff9b72be5e9f8b02e0b374c7c5c833
Last version change: 2021-05-18T17:38:58+0200

### PR DESCRIPTION
**Pull request by jpidelatorre**
- Added OSMOpeningHours to PointFeatureMarker to show formatted open hours.

closes #469

### CODE REVIEW

**Comment by leonardehrenfried** [comment]
Screenshot from 2021-05-21 09-30-55

This doesn't seem to work because the content has a different format. Can you just make text a bit smaller, please? No need to be perfect just a little less visually jarring.

**Comment by jpidelatorre** [comment]
@leonardehrenfried I think it works most of the time.

image

I'll make the text smaller too.

**Comment by jpidelatorre** [comment]
It also works with bike infrastructure.

image

**Comment by jpidelatorre** [comment]
@leonardehrenfried The font size is now smaller. Let me know if I should reduce the size of it.

**Comment by leonardehrenfried** [comment]
@jpidelatorre The problem is not the the font size of the OSMOpeningHours component but that the data is completely wrong as the content of the GeoJSON layer is different.

Compare:

image

with:

Screenshot from 2021-05-25 13-04-53

So the format is simply not compatible with the component.

For this reason, please completely stop using OSMOpeningHours and simply *reduce the font of the existing text*. Please don't change *anything else*.

**Comment by MartinH-open** [comment]
On the example of "Nette Toilette" this is strange:
Searching for "Nette Toilette" in all of Herrenberg in OSM reports the first instance near Würzburg.
None of the StadtNavi "Nette Toilette" ist visible in OSM (for me). I wonder where this comes from and if this is a reason for the reported problems here ?

**Comment by leonardehrenfried** [comment]
You want to use the following query to find them: https://overpass-turbo.eu/s/17IJ

**Comment by MartinH-open** [comment]
Thanks @leonardehrenfried 
I see now these are part of the attributes ( sorry - I did not know this )
In other areas these are added as extra entities "toilet" on the map  - perhaps some inconsistency in OSM 

```
Tags 15
addr:city=Herrenberg
addr:housenumber=1
addr:postcode=71083
addr:street=Marktplatz
changing_table=yes
name=Bürgeramt
office=government
opening_hours=Mo-We 08:30-17:00; Th 08:30-18:30; Fr 08:30-12:00; Sa 09:30-12:00
toilets=yes
toilets:female=yes
toilets:male=yes
toilets:scheme=Nette Toilette
toilets:wheelchair=yes
```

File: app/component/map/PointFeatureMarker.js [hunk 1]
```diff
@@ -179,7 +179,11 @@ const PointFeatureMarker = ({
           header={useDescriptionAsHeader ? description : header}
           subHeader={useDescriptionAsHeader ? '' : description}
         />
-        {popupContent && <div className="card-text">{popupContent}</div>}
+        {popupContent && (
+          <div className="location-popup-wrapper">
```
Comment (jpidelatorre) [inline app/component/map/PointFeatureMarker.js:183]: Added the same padding present in the opening hours table.

**Comment by jpidelatorre** [comment]
@leonardehrenfried I changed back the table and reduced the text. I'm seeing 20 files changed, but I only modified 2. Was there any revert in the `next` branch?

The files I changed are:
```
app/component/map/PointFeatureMarker.js
app/component/map/popups/marker-popup.scss
```

**Comment by leonardehrenfried** [comment]
No there wasn't. Please start a new branch, use `git cherry-pick` to move your commits to this branch, create a new PR and delete this one.

**Pull request closed without merging**